MakiseSharpServer.Identity/Views/Shared/_Layout.cshtml

```diff
@@ -1,25 +1,29 @@
-﻿@using System
+@using System
 @using System.Linq
 @using IdentityServer4.Extensions
 @using MakiseSharpServer.Application.Settings
 @inject AppSettings Settings
 @{
     string name = null;
     Uri avatar = null;
+    string tag = null;
     if (!true.Equals(ViewData["signed-out"]) && Context.User != null)
```

Comment:
```suggestion
    if (Context.User.IsAuthenticated())
```
it would throw an exception when eg. viewing an error page and not being signed-in